packages/@react-aria/interactions/docs/usePress.mdx

```diff
@@ -112,7 +112,11 @@ function Example() {
         tabIndex={0}>
         Press me!
       </div>
-      <ul>
+      <ul
+        style={{
```

Comment:
should it have a class name to do this since its repeated 3 times?

Reply:
if it was in our codebase, yes but I like that you see all the code required to make the example in the docs.

Reply:
oh great point!